pyvista/plotting/renderer.py

```diff
@@ -872,48 +873,89 @@ def add_actor(
 
         return actor, prop
 
-    def add_axes_at_origin(
+    def add_axes_at_origin(self, *args, **kwargs):  # numpydoc ignore=PR01,RT01
+        """Add axes actor at the origin.
+
+        This method is deprecated. Use `add_axes_marker` instead.
+        """
+        warnings.warn(
+            "Use of `add_axes_at_origin` is deprecated. Use `add_axes_marker` instead.",
+            PyVistaDeprecationWarning,
+        )
```

Comment:
Please add a comment in the form of: `# deprecated 0.43.0, convert to error in 0.45.0, remove 0.46.0`

This will help us grep for and follow through with deprecations during releases

See https://github.com/pyvista/pyvista/blob/main/CONTRIBUTING.rst#deprecating-features-or-other-backwards-breaking-changes